databricks-iris/{{ cookiecutter.repo_name }}/conf/base/catalog.yml

```diff
@@ -56,43 +56,43 @@ example_iris_data:
 # for all SparkDataSets.
 X_train@pyspark:
   type: spark.SparkDataSet
-  filepath: /dbfs/root/projects/iris-databricks/data/02_intermediate/X_train.parquet
+  filepath: /dbfs/FileStore/iris-databricks/data/02_intermediate/X_train.parquet
   save_args:
     mode: overwrite
 
 X_train@pandas:
   type: pandas.ParquetDataSet
-  filepath: /dbfs/root/projects/iris-databricks/data/02_intermediate/X_train.parquet
+  filepath: /dbfs/FileStore/iris-databricks/data/02_intermediate/X_train.parquet
 
 X_test@pyspark:
   type: spark.SparkDataSet
-  filepath: /dbfs/root/projects/iris-databricks/data/02_intermediate/X_test.parquet
+  filepath: /dbfs/FileStore/iris-databricks/data/02_intermediate/X_test.parquet
   save_args:
     mode: overwrite
 
 X_test@pandas:
   type: pandas.ParquetDataSet
-  filepath: /dbfs/root/projects/iris-databricks/data/02_intermediate/X_test.parquet
+  filepath: /dbfs/FileStore/iris-databricks/data/02_intermediate/X_test.parquet
 
 y_train@pyspark:
   type: spark.SparkDataSet
-  filepath: /dbfs/root/projects/iris-databricks/data/02_intermediate/y_train.parquet
+  filepath: /dbfs/FileStore/iris-databricks/data/02_intermediate/y_train.parquet
   save_args:
```

Comment:
This should be identical to `pyspark-iris`'s `databricks` folder right? Do we still need to keep the `databricks` folder in pyspark-iris? I think we should remove it if we are introducing a dedicated starter

Reply:
As I understood DBFS better, I realised we shouldn't be using `/dbfs/root`.

I have removed the databricks environment and all other databricks-related changes from `pyspark-iris` in this branch, I'm not sure why GitHub is not showing it as a diff.